src/Take.Blip.Builder.UnitTests/AttendanceRedirectHandlerTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Take.Blip.Builder.UnitTests
+{
+    public class AttendanceRedirectHandlerFixture //: IDisposable
+    {/*
+        public IContext Context { get; private set; }
```

Comment:
Do not commit "commented" code. Uncomment or remove it.

Reply:
The code its not like this. It was simply done for test one of the check steps. I'll rever this comment code.